docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
+. Click *Bar vertical stacked* to open the chart switcher
 
-. From the *Legend* dropdown, select the top position.
+. Click *Table*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_5.png[legend position]
+image::images/lens_end_to_end_2_1_2.png[Table with top values of request.keyword by most unique visitors]
 
-. Click *Save and return*.
+Customize the table even more:
 
-[discrete]
-[[percentage-stacked-area]]
-=== View the health of your website 
+. Click *Top values of request.keyword*
 
-To detect unusual traffic, bad website links, and server errors, create a percentage stacked area chart that displays the associated response codes.
+.. Increase the *Number of values*. The maximum allowed value is 1000.
 
-. Open *Lens*.
+.. In the *Display name* field, enter `Page URL`.
 
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
+. Click *Close*.
 
-.. Drag and drop *Records* to the *Vertical axis* field.
+. Click *Save and return*.
 
-.. Drag and drop *@timestamp* to the *Horizontal axis* field.
+. Arrange the table panel so that it has its own row, but keep the default width of half the screen.
 
-. From the *Chart Type* dropdown, select *Percentage bar*.
-
-. To remove the vertical axis label, click *Left axis*, then deselect *Show*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_4_3.png[Turn off axis name]
+. You do not need a panel title because the table columns are clearly labeled.
 
 [discrete]
-[[add-the-response-code-filters]]
-==== Add the response code filters
-
-For each response code that you want to display, create a filter. 
-
-. In the editor, click the *Drop a field or click to add* field for *Break down by*.
-
-. From *Select a function*, click *Filters*.
-
-. Add the filter for the successful response codes. 
-
-.. Click *All records*.
-
-.. In the *KQL* field, enter `response.keyword>=200 AND response.keyword<300`. 
-
-.. In the *Label* field, enter `2XX`, then press Return.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_4_1.png[First filter in filters aggregation]
-
-. Add the filter for the redirect codes. 
-
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=300 AND response.keyword<400`. 
-
-.. In the *Label* field, enter `3XX`, then press Return.
-
-. Add the filter for the client error codes. 
-
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=400 AND response.keyword<500`. 
-
-.. In the *Label* field, enter `4XX`, then press Return.
+[[custom-ranges]]
+=== Compare the percentage of a subset to the whole
 
-. Add the filter for the server error codes. 
+To compare a subset to the whole you need to create non-overlapping sets of documents that add up to 100%.
+For numeric data the *intervals* function is the best option, while for all other data types *filters* is
```

Comment:
Why? If this tutorial is for beginner users, do they care or know what the different functions mean/are?

Reply:
Teaching users when to use certain functions is the most important part of this tutorial, because it is the thing that they can most easily use on their own data.